package.json

```diff
@@ -2,12 +2,11 @@
   "name": "collection",
   "version": "1.0.0",
   "description": "Utility data structure used in Discord.js",
-  "main": "./src/index.js",
-  "types": "./typings/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "test": "npm run lint && npm run lint:typings && npm run unit-test",
-    "lint": "eslint ./",
-    "lint:typings": "tslint typings/index.d.ts",
+    "test": "npm run lint && npx tsc && npm run unit-test",
```

Comment:
Adding to this, you can add `rimraf` as a devDependency to clear out `dist/` before building again to make sure no artefacts are leftover in case of future additions/deletions/renames.

We could also improve on those annoying one liners by splitting them up and make use of the `pre` keyword to make them run before.

```json
"scripts" {
	"lint": "eslint src --ext .ts",
	"prebuild": "npm run lint",
	"build": "rimraf dist/ && tsc",
	"pretest": "npm run build",
	"test": "node test/index.js"
}
```

Runnig `npm test` or `yarn test` would result in `lint -> build -> test`